interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
-Your answer:
+Your answer: Yarn is a package manager that was created in 2016, yarn also adds node_ modules, yarn.lock. and package json.
 
-Researched answer:
+Researched answer: Yarn is a package manager for Javascript, that was created by Facebook in 2016. Yarn replaces the work for npm (largest software library (super cool to know!)) client or other package managers while remaining compatible with the npm registry. Yarn places dependencies locally were as npm pulls dependencies from the npm registry during every npm install. Yarn modifies in react by adding additional files such as node_modules, yarn.lock, and package j.son.
 
 3. What is an anonymous function in JavaScript?
 
-Your answer:
+Your answer: Lets start off by describing just a regular function, a function is created to preform a task. In order for the task to be performed, the function has to be invoked by something using "()" and can be invoked multiple times, while an anonymous doesn't need to be declared. 
 
-Researched answer:
+Researched answer: it is a nameless function that has the purpose of being accepted in multiple times but only has a single output varible .
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: both are similar with the fact that they both are js objects except props can get passed into a a component  where a state is based only in a component
 
-Researched answer:
+Researched answer: both are objects and hold information that determine functionality. Props can be passed into componets that is similar to a function, whereas state resides within a component where it is controlled 
 
 5. What is the DOM?
 
-Your answer:
+Your answer: document object model 
 
-Researched answer:
+Researched answer:DOM is an aplication programing interfaces for html and xml documents
 
 6. STRETCH: Which is the difference between a div and a span?
 
-Your answer:
+Your answer: Div allows a developer to tie in css into the main code, as for span, Im not quite sure to be exact. Span can also be contained in an html file but I will definitely utilize all my tool and resources in order to obtain more information about it
 
-Researched answer:
+Researched answer: A span is similar in a  function to div, they both are used to group information within an  HTML file. Div tags allow us to utilize CSS styling in the webpage, while span allows us to apply different text stylings.
 
 ## Looking Ahead: Terms for Next Week
 
-1. Object-oriented programming:
+1. Object-oriented programming: propgraming that focuses on objects and the data that the objects hold
 
-2. Ruby:
+2. Ruby:programing lange that focuses on simplicity and productivity
 
-3. Implicit return:
+3. Implicit returns: last executed  methods returns a value w/out being called.
 
-4. Ruby blocks:
+4. Ruby blocks: they are anonymous functions that can be passed into methods
 
-5. Ruby hashes:
+5. Ruby hashes: key value pairs
```

Comment:
Good work, Christian!  I can tell you're getting the concepts.  I recommend just studying up on your vocab and making your answers more conversational, like you did with #1 and #5.